api/specs.proto

```diff
@@ -103,6 +103,7 @@ message GlobalService {
 
 message TaskSpec {
 	oneof runtime {
+		NetworkAttacherSpec attacher = 8;
```

Comment:
I feel a little uncomfortable to put `NetworkAttacherSpec` as a `runtime`. Isn't it part of container runtime?